api/src/controllers/reviews.ts

```diff
@@ -95,7 +94,7 @@ router.get('/', async function (req, res) {
     courseID,
     professorID,
     userID,
-    _id: reviewID === undefined ? undefined : new ObjectId(reviewID),
+    _id: reviewID === undefined ? undefined : reviewID,
```

Comment:
not sure why this is a ternary condition as it functionally does nothing
```suggestion
    _id: reviewID,
```